src/config/chain.d.ts

```diff
@@ -26,6 +26,19 @@ export const CHAIN_ID: Record<ChainName, ChainId> = {
   VOLTA: '73799',
 }
 
+export const CHAIN_PREFIXES: Record<string, ChainId> = {
+  eth: CHAIN_ID.ETHEREUM,
+  rin: CHAIN_ID.RINKEBY,
+  bnb: CHAIN_ID.BSC,
+  xdai: CHAIN_ID.XDAI,
+  matic: CHAIN_ID.POLYGON,
+  ewt: CHAIN_ID.ENERGY_WEB_CHAIN,
+  arb1: CHAIN_ID.ARBITRUM,
+  vt: CHAIN_ID.VOLTA,
+  oeth: CHAIN_ID.OPTIMISM,
+  Avalanche: CHAIN_ID.AVALANCHE,
+}
```

Comment:
Ideally we should take this from the remote config. But that would require a refactoring that would wait for chains to be loaded before initializing the default id.